x-pack/plugins/ml/common/types/categories.ts

```diff
@@ -53,3 +54,49 @@ export interface FieldExampleCheck {
   valid: CATEGORY_EXAMPLES_VALIDATION_STATUS;
   message: string;
 }
+
+export const VALIDATION_CHECK_DESCRIPTION = {
+  [VALIDATION_RESULT.TOKEN_COUNT]: i18n.translate(
+    'xpack.ml.models.jobService.categorization.messages.validTokenLength',
+    {
+      defaultMessage:
+        'More than 3 tokens per example were found in over 75% of the examples loaded',
+    }
+  ),
+  [VALIDATION_RESULT.MEDIAN_LINE_LENGTH]: i18n.translate(
+    'xpack.ml.models.jobService.categorization.messages.validMedianLineLength',
+    {
+      defaultMessage: 'The median line length of the examples loaded was less than 400 characters.',
+    }
+  ),
+  [VALIDATION_RESULT.NULL_VALUES]: i18n.translate(
+    'xpack.ml.models.jobService.categorization.messages.validNullValues',
+    {
+      defaultMessage: 'Less than 25% of the examples loaded were null',
+    }
+  ),
+  [VALIDATION_RESULT.NO_EXAMPLES]: i18n.translate(
```

Comment:
Do we control the order these messages appear? For me, it would make sense to put this one first, followed by `The examples loaded were tokenized successfully.`

Reply:
yes, we can change the order.
They're currently ordered based on the `VALIDATION_RESULT` enum index.
i'll move that one, but is there a preferred order for all of the messages?